### PR TITLE
Added work section

### PR DESCRIPTION
**Pull request by kemar-art**
The second page was added to my portfolio project.
Content
- HTML
- CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,4 +41,29 @@ <h2>I’am a Software Developer</h2>
                 <li><a href="#" class="icon5"><i class="fab fa-medium-m fa-2x"></i></a></li>
             </ul>
             </div> 
-    </section>
\ No newline at end of file
+    </section>
+    <section>
+      <div>
+          <h1>My Recent Work</h1>
+      </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Good job using an `hx` tag here👍, kindly make sure you use `h2-h6` tag as you should have only one `h1` on your page so the `search engine` can find it easily 😉

File: index.html [hunk 2]
```diff
@@ -41,4 +41,29 @@ <h2>I’am a Software Developer</h2>
...
+      <section class="button">
+              <button type="submit" value="Submit" class="btn-icon1">css</button>
+              <button type="submit" value="Submit" class="btn-icon2">html</button>
+              <button type="submit" value="Submit" class="btn-icon3">bootstrap</button>
+              <button type="submit" value="Submit" class="btn-icon4">Ruby</button>
+          </section>
+          <section class="view">
+          <button type="submit" value="Submit" class="see-project">See Project</button>
+      </section>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Kindly make sure your design matches the one on your preferred template✌
- Your design
screenshot (47)
- Figma design
screenshot (48)

Comment (rotimiazeez) [inline index.html]: - [x] Kindly complete the rest of the project as seen in the template and project requirement✔
- Add the remaining work cards
screenshot (45)
- Add the about page
 screenshot (46)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @kemar-art 👋, 
## Good job so far!👏
### There are some issues that you still need to work on to go to the next project but you are almost there!😉
> Kindly complete the rest of the project as per the project requirement👍 so you can continue to the next project

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (kemar-art) [reply index.html]: This was done

File: README.md [hunk 3]
```diff
@@ -2,9 +2,7 @@
 
 # My Portfolio
 
-> This is the first project of my portfolio. It has no functionally as yet.
-
-> This project was to show my knowledge of what I have learned on HTML, CSS, and using flexbox.
+> I have now added the second page to my portfolio project, on this page my proficiency will desplay. But still no functionality added to the page as yet.
```
Comment (andersonlebon) [inline README.md]: ## [OPTIONAL] 
- [x] To make your README file more professional I would suggest adding a screenshot of the project in it.

**Review by andersonlebon — APPROVED** [review]
## Hi @kemar-art 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub flow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your design looks good :+1: :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**